packages/loader/src/gltf/extensions/index.ts

```diff
@@ -16,4 +16,6 @@ import "./GALACEAN_materials_remap";
 import "./GALACEAN_animation_event";
 
 export { GLTFExtensionParser, GLTFExtensionMode } from "./GLTFExtensionParser";
+export type { IGLTFExtensionVariants } from "./KHR_materials_variants";
```

Comment:
GLTFResource.extensionsData use this? but is any

Reply:
done